pkgs/by-name/mt/mtk_uartboot/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mtk-uartboot";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "981213";
+    repo = "mtk_uartboot";
+    rev = "v${version}";
+    hash = "sha256-mwwm2TVBfOEqvQIP0Vl4Q2SkcZxX1JP7rShmjaY+pWE=";
+  };
+
+  cargoHash = "sha256-At+HkvHcn6kAkSxBqFlaxUTZ7Xuy2lARMFQ2i6uybSk=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.IOKit
+  ];
+
+  meta = with lib; {
```

Comment:
- #208242
```suggestion
  meta = {
```

Reply:
I don't think usage of `with lib;` in Meta is "overuse".